packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -95,6 +99,21 @@ export const HelpCenterMoreResources = () => {
 						</a>
 					</div>
 				</li>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<a
+							href={ `https://wordpress.com/home/${ siteSlug }?myHomeCoursePaymentsModal=payments-features` }
+							rel="noreferrer"
+							target="_blank"
+							className="inline-help__video"
+							onClick={ () => trackMoreResourcesButtonClick( 'payments-features-video' ) }
+						>
+							<Icon icon={ video } size={ 24 } />
+							<span>{ __( 'Payments Features Videos', __i18n_text_domain__ ) }</span>
```

Comment:
I have not been able to test this yet, but I think this is adding a new item to the "more resources" section of the help center.

The section currently displays these 5 items:

<img width="402" alt="Screen Shot 2022-09-28 at 11 48 59" src="https://user-images.githubusercontent.com/1233880/192993581-66f65675-e6ed-415a-b429-e6627f15326a.png">

They all look quite generic to me, so I'm not sure if this is the best fit for something much more specific like videos payments features.

Reply:
@Automattic/vertex @jordesign @sethmclellana8c can you advise on this?

Reply:
For reference, this is how the new item looks like:

<img width="429" alt="Screen Shot 2022-09-29 at 15 37 03" src="https://user-images.githubusercontent.com/1233880/193046533-0f89726e-6b7e-4259-ad3c-dcd0cb98e958.png">

Reply:
Pinging @Automattic/vertex @jordesign @sethmclellana8c again, in case they missed my comment above.